### PR TITLE
[zh] translate concepts/Service Internal Traffic Policy

### LINKED ISSUE #27514: [zh] translate concepts/Service Internal Traffic Policy
**This is a Feature Request**
translate /docs/concepts/services-networking/service-traffic-policy.md into Chinese

**What would you like to be added**
content/zh/docs/concepts/services-networking/service-traffic-policy.md

/assign
/language zh

### PR DESCRIPTION
**Pull request by zhiguo-lu**
translate /docs/concepts/services-networking/service-traffic-policy.md into Chinese.

close #27514

### CODE REVIEW

**Comment by zhiguo-lu** [comment]
https://deploy-preview-27583--kubernetes-io-master-staging.netlify.app/zh/docs/concepts/services-networking/service-traffic-policy/

File: content/zh/docs/concepts/services-networking/service-traffic-policy.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+来为它指定一个内部专用的流量策略。
+此设置就相当于告诉 kube-proxy 对于集群内部流量只能使用本地的服务端口。
+
+<!-- 
+For pods on nodes with no endpoints for a given Service, the Service
+behaves as if it has zero endpoints (for Pods on this node) even if the service
+does have endpoints on other nodes.
+-->
+{{< note >}}
+如果该节点上的 Pod 均不提供指定 Service 的服务端点，
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/service-traffic-policy.md:58]:
```suggestion
如果某节点上的 Pod 均不提供指定 Service 的服务端点，
```

File: content/zh/docs/concepts/services-networking/service-traffic-policy.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+即使该 Service 在其他节点上有可用的服务端点，
+Service 的行为看起来也像是它只有 0 个服务端点（只针对此节点上的 Pod）。
+{{< /note >}}
+
+<!-- 
+The following example shows what a Service looks like when you set
+`.spec.internalTrafficPolicy` to `Local`:
+-->
+以下示例展示了当把 Service 的 `.spec.internalTrafficPolicy` 项设为 `Local` 时，
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/service-traffic-policy.md:67]:
```suggestion
以下示例展示了把 Service 的 `.spec.internalTrafficPolicy` 项设为 `Local` 时，
```

File: content/zh/docs/concepts/services-networking/service-traffic-policy.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+Service 的外观：
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/service-traffic-policy.md:68]:
```suggestion
Service 的样子：
```

File: content/zh/docs/concepts/services-networking/service-traffic-policy.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+  换句话说，在一个集群的不同 Service 上可以同时使用这两个特性，但在一个 Service 上不行。
+
+## {{% heading "whatsnext" %}}
+
+<!-- 
+* Read about [enabling Topology Aware Hints](/docs/tasks/administer-cluster/enabling-topology-aware-hints)
+* Read about [Service External Traffic Policy](/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip)
+* Read [Connecting Applications with Services](/docs/concepts/services-networking/connect-applications-service/)
+-->
+* 请阅读 [启用拓扑感知提示](/zh/docs/tasks/administer-cluster/enabling-topology-aware-hints)
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/service-traffic-policy.md:126]:
```suggestion
* 请阅读[启用拓扑感知提示](/zh/docs/tasks/administer-cluster/enabling-topology-aware-hints)
```

File: content/zh/docs/concepts/services-networking/service-traffic-policy.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+* 请阅读 [Service 的外部流量策略](/zh/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip)
+* 请阅读 [用 Service 连接应用](/zh/docs/concepts/services-networking/connect-applications-service/)
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/service-traffic-policy.md:128]:
```suggestion
* 请阅读[用 Service 连接应用](/zh/docs/concepts/services-networking/connect-applications-service/)
```

**Review by tanjunchen — COMMENTED** [review]
/lgtm

**Pull request merged**